pkg/scheduler/framework/interface.go

```diff
@@ -820,10 +819,9 @@ type Handle interface {
 
 	SharedInformerFactory() informers.SharedInformerFactory
 
-	// ResourceClaimCache returns an assume cache of ResourceClaim objects
-	// which gets populated by the shared informer factory and the dynamic resources
-	// plugin.
-	ResourceClaimCache() *assumecache.AssumeCache
```

Comment:
Since this method was added in 1.31 by #124595, are we sure we can remove this in 1.32?

Reply:
Good point, thanks for bringing this up.

I assumed so since it was just added specifically for the DRA plugin, if we leave it won't be used by any in-tree plugins. I guess in theory we could have out-of-tree plugins that started to depend on it right as it was introduced?

@kubernetes/sig-scheduling-leads Could you provide guidance here?

Reply:
@dom4ha @alculquicondor Any idea here? Is this ok?

Reply:
Ack, thanks for the confirmation!